src/lib/homestore_backend/resync_blob_data.fbs

```diff
@@ -0,0 +1,16 @@
+namespace homeobject;
+
+table BlobData {
+    shard_id : uint64;
```

Comment:
nit: may be uint32 can be together so that they are packed. shard_id, blob_id, data_size, user_key_size, data, user_key

Reply:
sure. will do